app/view_models/nsm/v1/defendant_details.rb

```diff
@@ -29,45 +29,51 @@ def main_defendant_maat
         main_defendant['maat']
       end
 
+      def main_defendant_value
+        if main_defendant_maat.present?
```

Comment:
You could probably DRY this up by doing `construct_value(defendants.find { _1['main'] })`

Reply:
updated